src/main/java/com/eternalcode/combat/drop/DropModifier.java

```diff
@@ -0,0 +1,14 @@
+package com.eternalcode.combat.drop;
+
+import com.eternalcode.combat.config.implementation.PluginConfig;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.List;
+
+public interface DropModifier {
+
+    DropType getDropType();
+
+    void modifyDrop(DropInfo info, PluginConfig config);
```

Comment:
nie przekazuj PluginConfig przez metodę w interfejsie, możesz to zrobić w implemntacji w konstruktorze